cmd/admin-handlers-config-kv.go

```diff
@@ -518,3 +537,18 @@ func (a adminAPIHandlers) GetConfigHandler(w http.ResponseWriter, r *http.Reques
 
 	writeSuccessResponseJSON(w, econfigData)
 }
+
+// setLoggerWebhookSubnetProxy - Sets the logger webhook's subnet proxy value to
+// one being set for subnet proxy
+func setLoggerWebhookSubnetProxy(subSys string, cfg config.Config) bool {
+	if subSys == config.SubnetSubSys {
+		loggerWebhookSubnetProxy := cfg[config.LoggerWebhookSubSys][config.SubnetSubSys].Get(logger.Proxy)
+		subnetProxy := cfg[config.SubnetSubSys][config.Default].Get(logger.Proxy)
+		if loggerWebhookSubnetProxy != subnetProxy {
+			obj := cfg[config.LoggerWebhookSubSys][subnet.LoggerWebhookName]
+			obj.Set(logger.Proxy, subnetProxy)
+			return true
+		}
```

Comment:
```suggestion
                subnetWebhookCfg := cfg[config.LoggerWebhookSubSys][subnet.LoggerWebhookName]
		loggerWebhookSubnetProxy := subnetWebhookCfg.Get(config.Proxy)
		subnetProxy := cfg[config.SubnetSubSys][config.Default].Get(config.Proxy)
		if loggerWebhookSubnetProxy != subnetProxy {
			subnetWebhookCfg.Set(logger.Proxy, subnetProxy)
			return true
		}
```

- simplify by capturing `cfg[config.LoggerWebhookSubSys][subnet.LoggerWebhookName]` inside a variable (`subnetWebhookCfg`) and using it in both places
- Use `config.Proxy` instead of `logger.Proxy`. That latter can be removed.

Reply:
Ack